tamr_unify_client/_custom_encoder.py

```diff
@@ -0,0 +1,89 @@
+# flake8: noqa: C901
+"""Adaptation of the Python standard library JSONEncoder to encode `NaN` as 'null'
+Compare to https://github.com/python/cpython/blob/3.9/Lib/json/encoder.py
+The only functional difference is in the definition of `floatstr` where 'NaN', 'Infinity', and '-Infinity' are encoded as 'null'
+"""
+from json import JSONEncoder
+from json.encoder import (
+    _make_iterencode,
+    py_encode_basestring,
+    py_encode_basestring_ascii,
+)
+
+try:
+    from _json import encode_basestring_ascii as c_encode_basestring_ascii
+except ImportError:
+    c_encode_basestring_ascii = None
+try:
+    from _json import encode_basestring as c_encode_basestring
+except ImportError:
+    c_encode_basestring = None
+try:
+    from _json import make_encoder as c_make_encoder
+except ImportError:
+    c_make_encoder = None
+
+INFINITY = float("inf")
+encode_basestring = c_encode_basestring or py_encode_basestring
+encode_basestring_ascii = c_encode_basestring_ascii or py_encode_basestring_ascii
+
+
+class CustomEncoder(JSONEncoder):
```

Comment:
Instead of `CustomEncoder` how about something like `IgnoreNanEncoder`?

```suggestion
class IgnoreNanEncoder(JSONEncoder):
```

Reply:
Done.